kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/modem/ModemDriver.java

```diff
@@ -93,32 +96,32 @@
                 String gpioPath = BASE_GPIO_PATH + gpioIndex;
```

Comment:
Do you think we can try to externalize the block of codes that start at line 93 (96 of the new code), 103 (106) and and 114 (117) into a single method with a parameter (the offset)?